src/webhooks.ts

```diff
@@ -40,6 +40,18 @@ export class Webhooks {
       url: `/forms/${uid}/webhooks`,
     })
   }
+  
+  public toggle (args: { uid: string, tag: string, enabled: boolean }): Promise<Typeform.API.Webhooks.List> {
+    const { uid, tag, enabled } = args
```

Comment:
I tested this and it works nicely. However if the `enabled` is not provided we assume `false` value without any further info.

I am thinking about throwing an error when the value is not provided - the API call does not make much sense without it, right? However other API calls do not throw when params are missing either, even if they fail. 

Maybe we could address this in as a separate feature / PR. WDYT @picsoung ?

Reply:
Thanks for your work man!
We should definitively take advantage to check the params shape, but across all endpoints, before it hits the API.